client/Scenarios/GetVMTaskExamples.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Azure.ResourceManager.Core;
+using Azure.ResourceManager.Network;
+using azure_proto_compute;
+using azure_proto_network;
+using System;
+using System.Linq;
+
+namespace client
+{
+    class GetVMTaskExamples : Scenario
+    {
+        public GetVMTaskExamples() : base() { }
+
+        public GetVMTaskExamples(ScenarioContext context) : base(context) { }
+
+        public override void Execute()
+        {
+            var client = new AzureResourceManagerClient();
+
+            string vmId = "/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c/resourceGroups/nbhatia-test/providers/Microsoft.Compute/virtualMachines/nibhati";
```

Comment:
update to not be hard coded ids since we cannot guarantee these will exist forever